examples/arf-pid/specification.yml

```diff
@@ -37,8 +37,8 @@ user_claims:
 holder_disclosed_claims:
   nationalities:
     - true
-  age_equal_or_over:
-    '18': true
+  #age_equal_or_over:
```

Comment:
We should probably chase this down sometime sooner rather than later (some issue with the yaml or a bug in the library maybe?).

Reply:
created "disclosing age_equal_or_over not working" #409